working/templates/automationsolution/.template_config/template.json

```diff
@@ -24,18 +24,10 @@
     "Author": {
           "type": "parameter",
           "datatype": "string",
-          "defaultValue": "SKYLINE2\\XXX",
+          "defaultValue": "",
           "replaces": "$AUTHOR$",
-          "description": "The domain name of the author."
+          "description": "The author."
       },
-    "CopyrightYear": {
-      "type": "generated",
-      "generator": "now",
-      "replaces": "$COPYRIGHTYEAR$",
-      "parameters": {
-        "format": "yyyy"
-      }
-    },
     "InitialVersionDate": {
```

Comment:
I couldn't immediately find if it was still used, but is it still used? Otherwise it can also be removed like AuthorVersionHistory.

Reply:
Ignore this. As you said, it's used in the VersionHistory tags